shell/browser/ui/gtk_util.cc

```diff
@@ -14,30 +14,16 @@
 
 namespace gtk_util {
 
-// Copied from L40-L55 in
-// https://cs.chromium.org/chromium/src/chrome/browser/ui/libgtkui/select_file_dialog_impl_gtk.cc
-#if GTK_CHECK_VERSION(3, 90, 0)
-// GTK stock items have been deprecated.  The docs say to switch to using the
-// strings "_Open", etc.  However this breaks i18n.  We could supply our own
+// Currently i18n is broken for file dialog labels. We could supply our own
 // internationalized strings, but the "_" in these strings is significant: it's
-// the keyboard shortcut to select these actions.  TODO: Provide
+// the keyboard shortcut to select these actions. TODO: Provide
```

Comment:
This change seems unrelated to the native dialog patch.

This hack seems to [still exist in upstream Chromium](https://source.chromium.org/chromium/chromium/src/+/master:ui/gtk/select_file_dialog_impl_gtk.cc;l=37) so I've a slight preference to keeping consistency with upstream.

Also -- if I'm reading it right, this change seems to go from "here is an ugly hack to avoid breaking i18n on file dialog button labels on systems using GTK < 3.90" to "file dialog button labels are broken < 3.90". Is that correct?

Reply:
After some discussion in #gtk, I now understand the hack. Since GTK 4.0 has now been released, does it make sense to either update the CHECK_VERSION macro to reflect this, or B drop this ability to compile against 4.0 successfully. It seems a little weird to me, because I am not checking to make sure the new GTK code works against 4.0 unless you think I should do that.